src/atomate2/common/schemas/pheasy.py

```diff
@@ -505,16 +513,18 @@ def from_forces_born(
             num_judge = len(disps)
 
             if num_judge > 3:
-                pheasy_cmd_14 = 'pheasy --dim "{0}" "{1}" "{2}" -f --c2 10.0 --full_ifc -w 2 --symprec 1e-3 -l LASSO --std --rasr BHH --ndata "{3}"'.format(
+                pheasy_cmd_14 = 'pheasy --dim "{0}" "{1}" "{2}" -f --c2 10.0 --full_ifc -w 2 --symprec "{3}" -l LASSO --std --rasr BHH --ndata "{4}"'.format(
                     int(supercell_matrix[0][0]),
                     int(supercell_matrix[1][1]), 
-                    int(supercell_matrix[2][2]), 
+                    int(supercell_matrix[2][2]),
+                    float(symprec), 
                     int(num_har))
             else:
-                pheasy_cmd_14 = 'pheasy --dim "{0}" "{1}" "{2}" -f --full_ifc --c2 10.0 -w 2 --symprec 1e-3 --rasr BHH --ndata "{3}"'.format(
+                pheasy_cmd_14 = 'pheasy --dim "{0}" "{1}" "{2}" -f --full_ifc --c2 10.0 -w 2 --symprec "{3}" --rasr BHH --ndata "{4}"'.format(
```

Comment:
There seem to be more parameters here. Maybe, you can check the whole code again for such points. I might overlook parts here.

Reply:
Will check it.